src/main/java/org/folio/innreach/client/CirculationClient.java

```diff
@@ -56,4 +57,7 @@
   @PostMapping("/renew-by-id")
   LoanDTO renewLoan(@RequestBody RenewByIdDTO renewLoan);
 
+  @PostMapping("/loans/{loanId}/claim-item-returned")
```

Comment:
Have you added modulePermission to access this endpoint inside /d2ir/circ/*** call or it's already there?